HaskellMusicEngine.cabal

```
@@ -39,4 +44,6 @@ executable HaskellMusicEngine
     build-depends:
          base >=4.7 && < 5
       ,  codeworld-api
+      ,  HCodecs
+      ,  text == 1.2.4.1
```

As one of you suggested in review of another PR, let's have dependencies' versions specified properly
```suggestion
      ,  HCodecs >= 0.5.2 && < 0.6
      ,  text >= 1.2.4 && < 2
```